jina/peapods/runtimes/remote/jinad/api.py

```diff
@@ -253,3 +253,12 @@ class PeaAPI(JinadAPI):
 class PodAPI(JinadAPI):
     """Pod API, we might have different endpoints for peas & pods later"""
     kind = 'pod'
+
+
+def get_jinad_api(kind: str, host: str, port: int, logger: JinaLogger, **kwargs):
+    if kind == 'pea':
+        return PeaAPI(host=host, port=port, logger=logger, **kwargs)
```

Comment:
What about `PeaJinadAPI` and `PodJinadAPI`? 
```suggestion
        return PeaAPI(host=host, port=port, logger=logger, **kwargs)
```